Package.swift

```diff
@@ -690,6 +691,13 @@ let package = Package(
             ]
         ),
 
+        // MARK: C helpers
+
+        .target(
+            name: "CExecHelpers",
```

Comment:
Swift has a global namespace for module names, so I'd suggest to give this a slightly more unique name too.